conf/broker.conf

```diff
@@ -618,13 +618,15 @@ tlsCiphers=
 # authentication.
 tlsRequireTrustedClientCertOnConnect=false
 
+# Specify the TLS provider for the broker service:
```

Comment:
@momo-jun need to add explanations to https://pulsar.apache.org/docs/en/next/reference-configuration/?

Reply:
Yep, to both broker/standalone and proxy sections. Will add the docs after the 2.10 release.